app/controllers/pages_controller.rb

```diff
@@ -6,6 +6,22 @@ def show
   end
 
   def induction_tutor_materials
-    render template: "pages/induction_tutor_materials/#{params[:provider].tr('-', '_')}/#{params[:year].tr('-', '_')}"
+    provider_name = params[:provider]
+    render template: "pages/induction_tutor_materials/#{folder_name(provider_name)}/#{params[:year].tr('-', '_')}"
+  end
+
+private
+
+  def folder_name(provider_name)
+    case provider_name
+    when "ambition-institute"
+      "ambition"
+    when "educational-development-trust"
+      "educational_development_trust"
+    when "teach-first"
+      "teach_first"
+    when "ucl-institute-of-education"
+      "ucl"
+    end
```

Comment:
Should we rename the folders instead, so we can use `provider_name.underscore` for all of them?

Reply:
i was thinking to do that but thought it better to change code rather than folder names if providers ever change their names. Don't mind either way.